sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -113,6 +113,35 @@
         ]
       }
     },
+    "_oai": {
+      "description": "OAI-PMH specific information.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "id": {
+          "description": "OAI-PMH record identifier.",
+          "type": "string",
+          "minLength": 1
+        },
+        "sets": {
+          "description": "OAI-PMH sets for record.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          }
+        },
+        "updated": {
+          "description": "Last OAI-PMH set update (ISO8601 formatted timestamp)",
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "required": [
```

Comment:
For me, `updated` property should be required. When a document is created, then the `updated` field should be equal to creation date

Reply:
Absolutely right, corrected!